### PR TITLE
Task/63046 ProductのCRUD作成 /　Create Product CRUD

### PR DESCRIPTION
**Pull request by e346m**

### CODE REVIEW

File: app/models/product.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Product < ActiveRecord::Base
```
Comment (khanhhd) [inline app/models/product.rb:1]: Should we add reference for product, product belongs_to user and product belongs_to m_book ?

**Comment by norifumi** [comment]
LGTM

File: app/models/product.rb (hunk 1 above)
Comment (e346m) [inline app/models/product.rb:1]: right!! . product should have reference of m_book. but I'll implement request model so ref of user should be written in request!!

File: db/schema.rb [hunk 2]
```diff
@@ -20,6 +20,16 @@
...
+  create_table "products", force: :cascade do |t|
+    t.integer  "m_book_id",       limit: 4
+    t.integer  "status",          limit: 4
+    t.datetime "requested_date"
+    t.integer  "request_user_id", limit: 4
+    t.integer  "rental_user_id",  limit: 4
+    t.datetime "created_at",                null: false
+    t.datetime "updated_at",                null: false
+  end
+
```
Comment (e346m) [inline db/schema.rb:32]: I'll fix table on another ticket

File: app/models/product.rb (hunk 1 above)
Comment (khanhhd) [inline app/models/product.rb:1]: @eiji346 I got it

**Comment by norifumi** [comment]
LGTM

**Pull request merged**